### DISCUSSION TITLE
hikvision ax pro

### DISCUSSION THREAD

**Discussion by davidemarta0527** [petrleocompel/hikaxpro_hacs#92] [General]
Hi, I installed the hikvision ax pro integration, it reads all the sensors and the alarm correctly, but it doesn't let me activate the alarm because when I click on activate, it also invokes the service via central because I feel that they communicate but then on the hikvision app says insertion failed...I installed home assistant on minipc with haos...can you help me please?

**Comment by petrleocompel** [comment]
As I provided questions on kofi. 
I need you to check id your account is admin account. Then you can try to check logs after testing activating the alarm.

**Reply by davidemarta0527** [reply]
in the integration I entered the username and password of the administrator account, nothing is written in the registers

**Reply by petrleocompel** [reply]
I have no clue what type of HA installation you use. So I cannot fully navigate you how to enable logs. But there are "integration logs" (debug logs for integration) also in configuration of hikaxpro is settings to enable debug. (HA needs restart after that)
Are you using "pin" to activate the alarm ?

**Reply by davidemarta0527** [reply]
i use haos system , no pin for activate the alarm , debug activated...

**Reply by davidemarta0527** [reply]
if I activate the alarm from the hikconnect app I can deactivate it via home assistant, on the contrary I cannot activate it from home assistant

**Reply by petrleocompel** [reply]
Which device you have ? Also checkbox "Use code for arming" is unticked? 
can you screenshot your settings of the integration?

**Reply by davidemarta0527** [reply]
mobile device iphone..hikvision device is ax pro , Attached are photos of the hikvision device and configuration.

**Reply by petrleocompel** [reply]
There is more devices AX Pro ... But I guess it is wireless version of AX Pro with max 64 or 96 devices. 

Settings looks right. You can delete the picture. 

Sadly it seems like your user might does not have the permissions. Can you take screenshot of web ui of AX pro of user accouts and permissions. Provide by email to petrleocompel@gmail.com for security. Or mask the addresses for security.

**Reply by davidemarta0527** [reply]
tks , email sent

**Reply by petrleocompel** [reply]
We found out the reasons why it can have problem.

Added section to `README.md`. Please check this and retry. 

### Pre-check
⚠️ Please make sure your user you will be using will have role "Admin" and it is not same as "Installer" (same email address) or remove the "Installer" completely. (Referring to issue #108)
